index.js

```diff
@@ -1,18 +1,76 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
 
+window.onload = function getHeader() {
+    let header = document.body.querySelector("#header");
+    console.log("Here's your header:", header);
 
-/***** Deliverable 2 *****/
 
+    /***** Deliverable 2 *****/
 
+    header.style.backgroundColor = "orange";
+
+    /***** Deliverable 3 *****/
+    console.log('This is what the traveler object looks like: ', traveler);
+
+    let travelerName = document.body.querySelector("#profile h2");
+    travelerName.innerHTML = `${traveler.name}`;
+
+    let travelerPhoto = document.body.querySelector("#profile img");
+    travelerPhoto.src = `${traveler.photo}`;
+    travelerPhoto.alt = `${traveler.name}`;
+
```

Comment:
Good! 
- One recommendation I have is to choose ```textContent``` over ```innerHTML``` when possible especially if you know what you're inserting should be plain text. It's better practice for security reasons. You can read more about the security implications here: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations
- Another is that you don't have to use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) because you're not string interpolating.
```suggestion
    let travelerName = document.body.querySelector("#profile h2");
    travelerName.textContent = traveler.name;

    let travelerPhoto = document.body.querySelector("#profile img");
    travelerPhoto.src = traveler.photo;
    travelerPhoto.alt = traveler.name;

```